static/src/types.ts

```diff
@@ -0,0 +1,39 @@
+export interface ToastOptions {
+  type?: "success" | "danger" | "warning" | "info";
+  duration?: number;
+  dismissable?: boolean;
```

Comment:
Why are these all optional? Shouldn't they rather be required but provide defaults?